OmiseSDK/Client.swift

```diff
@@ -80,7 +80,7 @@ import os
             defer {
                 if #available(iOSApplicationExtension 10.0, *) {
                     switch result {
-                    case .success(let value):
+                    case .success(let _value):
```

Comment:
We can remove `(let value)`, just use `case .success:`

Reply:
Changed to `case .success:`